pandas/tests/extension/base/ops.py

```diff
@@ -69,7 +69,8 @@ def test_arith_series_with_array(self, data, all_arithmetic_operators):
         # ndarray & other series
         op_name = all_arithmetic_operators
         s = pd.Series(data)
-        self.check_opname(s, op_name, [s.iloc[0]] * len(s), exc=TypeError)
+        self.check_opname(s, op_name, pd.Series([s.iloc[0]] * len(s)),
```

Comment:
why this change?

Reply:
Similar as what Tom said below: `combine` doesn't work with lists (it might be we should improve `combine`, but that is a separate issue)